cine-flex/src/components/Button/Button.jsx

```diff
@@ -0,0 +1,27 @@
+import styles from '../Button/Button.module.scss';
```

Comment:
could you tell me why the relative URL is from folder instead of './filename'

Reply:
I had issues with auto-suggestion and it did not suggest the file name. So I moved to import from the folder. Will update it.